docs/language/lateral-subqueries.md

```diff
@@ -112,13 +126,13 @@ parenthesized form:
 ```
 ( over <expr> [, <expr>...] [with <var>=<expr> [, ... <var>[=<expr>]] | <lateral> )
 ```
-> Note that the parentheses disambiguate a lateral expression from a lateral
-> dataflow operator.
 
-This form must always include a lateral scope as indicated by `<lateral>`,
-which can be any dataflow operator sequence excluding [`from` operators](operators/from.md).
-As with the `over` operator, values from the outer scope can be brought into
-the lateral scope using the `with` clause.
+:::tip
+The parentheses disambiguate a lateral expression from a lateral
+dataflow operator.
+:::
```

Comment:
I'd appreciate some input on this text. All I've done with it thus far is reformat it. However, as a user/reader, I struggle to understand what it's trying to clarify. I'd be in favor of expanding the text a bit to make it clearer or drop it until some user's confusion reminds us why we put it there. 😄

Reply:
Update: I've spent more time staring at it and I think I at least understand what it's trying to say, but I'm still struggling for the text to propose instead. I _think_ the point being made is that the open parenthesis appearing _before_ the keyword `over` is what makes it clear that we're in expression context and not dataflow context.

The reason the text was bothering me is because, assuming a new user reading these docs for the first time, we've just showered them with a bunch of related terms on this page: Lateral Subquery, Lateral Scope, Lateral Expression... and it feels like in this one sentence we cooked up "lateral dataflow operator" as yet another, but unlike the others, it's not a specific term that's defined in some place we can link to.

I think I'm still leaning toward just taking it out. I'm feeling like the overall Lateral Subquery topic is one of those growth points as a user learning Zed where there's no shortcuts and they just need to closely read the docs, hack with the examples, apply it to their own use cases, and when they come out the other side it'll hopefully have stuck. And I feel like this particular point about the parens comes through already in the text without this additional note. But I'm open to other thoughts!

Reply:
I think this should stay and is fine as is. Rather than adding more text, how about just making "lateral dataflow operator" into a link to the `over` operator page?